packages/regl-worldview/src/commands/GLText.js

```diff
@@ -354,7 +363,19 @@ function makeTextCommand(alphabet?: string[]) {
         let y = 0;
         let markerInstances = 0;
 
-        const fgColor = marker.colors?.[0] || marker.color || BG_COLOR_LIGHT;
+        const hitmapColor = (markerColor) => {
```

Comment:
Would be good to have a type annotation for the argument here.

Reply:
While this code is outdate, I am use explicit types now. Let me know what you think.